src/components/ebay-chip/index.marko

```diff
@@ -0,0 +1,19 @@
+import { processHtmlAttributes } from "../../common/html-attributes";
+$ const { renderBody, deleteButtonLabel, ...htmlAttributes } = input;
+
+<span class="chip" ...processHtmlAttributes(htmlAttributes)>
+    <span class="chip__text" id:scoped="text">
+        <${input.renderBody}/>
```

Comment:
Since we have `renderBody` defined at the top (so it doesnt get added in spread), why not use it here? At least that way we don't have any unused variables